whisper/whisperv5/peer_test.go

```diff
@@ -159,6 +142,15 @@ func initialize(t *testing.T) {
 			t.Fatalf("failed to start server %d.", i)
 		}
 
+		if i > 0 {
+			peerNodeId := nodes[i-1].id
+			address, _ := net.ResolveTCPAddr("tcp", nodes[i-1].server.ListenAddr)
+			peerPort := uint16(address.Port)
+			peerNode := discover.PubkeyID(&peerNodeId.PublicKey)
+			peer := discover.NewNode(peerNode, address.IP, peerPort, peerPort)
+			node.server.AddPeer(peer)
```

Comment:
You're changing the behavior of the tests here: each node has only one peer (the one coming before) while bootstrap + static + trusted nodes were all previous nodes.